.goreleaser-release.yaml

```diff
@@ -0,0 +1,105 @@
+# This goreleaser config file requires GoReleaser Pro as it uses the prebuilt
+# builder type.
+
+archives:
+  - id: coder-linux
+    builds: [release-prebuilt-linux]
+    format: tar.gz
```

Comment:
What about Windows and Darwin archives? Are they still uploaded?

Reply:
I see below they are listed as extra files. Right now, the Windows and Darwin binaries are zipped up.

Reply:
I fixed the glob not using the zipped versions